.eslintrc.js

```diff
@@ -163,5 +163,13 @@ module.exports = {
                 ],
             },
         },
+        {
+            files: ['tests/**/*.{js,jsx,ts,tsx}', '.github/**/*.{js,jsx,ts,tsx}'],
+            rules: {
+                '@lwc/lwc/no-async-await': 'off',
+                'no-await-in-loop': 'off',
+                'no-restricted-syntax': ['error', 'ForInStatement', 'LabeledStatement', 'WithStatement'],
```

Comment:
All I'm doing with this statement is allowing this syntax:

```js
for (const pr of PRList) {
    ...
}
```

Which is very simple. I'm pretty sure the performance concerns from the AirBnb are way out of date but they still haven't updated their ESLint standard because they don't like loops in general. In the case of all our GitHub Actions, I think for-each loops like this are nice simplification because they allow us to do `await` inside the loop to wait for async stuff in one iteration to finish before moving on to the next.